CONTRIBUTING.md

```diff
@@ -1,57 +1,108 @@
+
+Wrap all Objective-C in `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END` blocks. Make sure to annotate any types that are nullable correctly.
 
+## Getting started
 Given that you have a compatible environment as stated above you can now setup the project.
 
 - `bundle install` to install ruby dependencies
 - `npm install` to install npm dependencies
-- If you work for Skyscanner, [configure relative](#relative-font) (optional).
 - `(cd Example && bundle exec pod install)` To setup the example project.
 - `open Example/Backpack.xcworkspace` to open the example project
 
-## Relative Font
+## Adding a new component
+If you want to add a new component, we will need the following:
 
-> Skyscanner employees only
+- Design (Figma file)
+- UIKit and SwiftUI component
+- Accessibility
+- Stories
+- Tests (unit & snapshot)
+- Documentation (Including main `REAMDE.md`)
 
-Our fonts can only be used by Skyscanner employees. If you don't work for Skyscanner don't worry - the Example app will still work just fine with iOS system font too!
+### Design
 
-To use our `Skyscanner Relative` font-face in the example app do the following:
+Figma is the preferred format for non-technical folks. We’d appreciate if you could provide an exact match of your component in Figma format together with folders for each state e.g. disabled, expanded etc.
 
-- Make sure you're connected to the VPN.
-- If you've already done a `pod install`, delete `Example/Pods`.
-- Set the environment variable using `export BPK_USE_RELATIVE=1`. (Put this in your `.bashrc`/`.zshrc` for convenience.)
-- Setup the project as [above](#setup).
+### UIKit and SwiftUI component
 
-During pod install, fonts will be downloaded and made available to the project automatically.
+Make sure that when you contribute you are considering a UIKit and a SwiftUI version of your component. The UIKit component should live in the Backpack folder. A SwiftUI component needs to be located in Backpack-SwiftUI.
+
+A component folder structure is setup as follows. Please take a look at the setup of existing components for examples. 
+
+* Backpack
+    - {ComponentName}
+        - Classes
+            - BPK{ComponentName.swift}
+            - {Any supporting swift files}
+        - README.md
+
+For SwiftUI the root folder will be `Backpack-SwiftUI`. The structure is otherwise the same.
+
+### Accessibility
+Our design system tries to deliver accessible components. When designing your component make sure the accessibility of your component is considered.
+
+At a mimimum you should consider the following
+
+* Colour contrast meets AA standard
+* Required accessibility labels are passed to the component
+* Screen readers can interact with the component
 
-## Testing
+### Stories
+Each component needs to be visually documented in our example app. Make sure to add a new entry and showcase your component in each of its different states and variants. 
```

Comment:
can you add how/where?

Reply:
Or I guess when the boilerplate component is made this will just be included with the framework setup?

Reply:
boilerplating takes care of this in the future.